rstream/producer.py

```diff
@@ -190,21 +226,54 @@ async def publish_batch(
 
         return publishing_ids
 
-    async def publish(
+    async def send_wait(
         self,
         stream: str,
         message: MessageT,
-        sync: bool = True,
         publisher_name: Optional[str] = None,
     ) -> int:
-        publishing_ids = await self.publish_batch(
+        publishing_ids = await self._send_batch(
             stream,
             [message],
-            sync=sync,
+            sync=True,
             publisher_name=publisher_name,
         )
         return publishing_ids[0]
 
+    async def send(
+        self,
+        stream: str,
+        message: MessageT,
+        publisher_name: Optional[str] = None,
+    ):
+
+        if stream not in self._buffered_messages:
+            self._buffered_messages[stream] = []
+        async with self._lock:
+            await self._get_or_create_publisher(
+                stream, send_batch_enabled=True, publisher_name=publisher_name
+            )
+        async with self._buffered_messages_lock:
+            self._buffered_messages[stream].append(message)
+
+        await asyncio.sleep(0)
+
+    # After the timeout send the messages in _buffered_messages in batches
+    async def _timer(self, timeout, sync):
```

Comment:
I think we can drop passing `timeout` and `sync` arguments here - `timeout` is stored in an attribute anyway and `sync` is always False here